src/cloudWatchLogs/registry/logStreamRegistry.ts

```diff
@@ -212,20 +194,29 @@ export async function filterLogEventsFromUriComponents(
 ): Promise<CloudWatchLogsResponse> {
     const client = new DefaultCloudWatchLogsClient(logGroupInfo.regionName)
 
-    const response = await client.filterLogEvents({
+    const timeout = new Timeout(10000)
```

Comment:
Let's just set it to something arbitrary like 5 minutes:
```suggestion
    const timeout = new Timeout(300000)
```